roles/openshift_logging_elasticsearch/tasks/restart_cluster.yml

```diff
@@ -1,91 +1,113 @@
 ---
-# Disable external communication for {{ _cluster_component }}
```

Comment:
This shouldn't have been removed... must have had a bad rebase that i didn't catch?
@sdodson i'll be opening up a follow up

Reply:
scratch that... i see now that it was moved into a block. ignore